Gordon360/Services/HousingService.cs

```diff
@@ -135,11 +123,65 @@ public int SaveApplication(int apartAppId, string editor, string sess_cde, strin
                 applicantResult = RawSqlQuery<ApartmentApplicantViewModel>.query("INSERT_AA_APPLICANT @APPLICATION_ID, @ID_NUM, @APRT_PROGRAM, @SESS_CDE", appIdParam, idParam, programParam, sessionParam); //run stored procedure
                 if (applicantResult == null)
                 {
-                    throw new ResourceNotFoundException() { ExceptionMessage = "Applicant with ID " + id + " could not be saved." };   
+                    throw new ResourceNotFoundException() { ExceptionMessage = "Applicant with ID " + id + " could not be saved." };
                 }
             }
 
             return appId;
         }
+
+        /// <summary>
+        /// Changes the student user who has permission to edit the given application
+        ///
+        /// </summary>
+        /// <returns>Whether or not all the queries succeeded</returns>
+        public bool ChangeApplicationEditor(int apartAppId, string editorId, string newEditorId)
+        {
+            IEnumerable<ApartmentAppEditorViewModel> editorResult = null;
+            IEnumerable<ApartmentAppSaveResultViewModel> result = null;
+
+            DateTime now = System.DateTime.Now;
+
+            SqlParameter appIdParam = null;
+            SqlParameter editorParam = null;
+            SqlParameter timeParam = null;
+            SqlParameter newEditorParam = null;
+
+            appIdParam = new SqlParameter("@APPLICATION_ID", apartAppId);
+
+            editorResult = RawSqlQuery<ApartmentAppEditorViewModel>.query("GET_AA_EDITOR_BY_APPID @APPLICATION_ID", appIdParam);
+            if (editorResult == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The application could not be found." };
+            }
+            else if (!editorResult.Any())
+            {
+                return false;
+            }
+
+            ApartmentAppEditorViewModel editorModel = editorResult.ElementAt(0);
+            string storedEditorId = editorModel.ModifiedBy;
```

Comment:
Update the "ModifiedBy" to "Editor".

Reply:
Does that mean we changed the name of the column in the database table?

Reply:
The change has been made

Reply:
Josh and I will right after this.